orc-rt/include/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ target_include_directories(orc-rt-headers INTERFACE
 set_property(TARGET orc-rt-headers
     PROPERTY PUBLIC_HEADER ${ORC_RT_HEADERS}
 )
-install(DIRECTORY ./
+install(TARGETS orc-rt-headers
     DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/
-    FILES_MATCHING PATTERN "*.h"
+    COMPONENT OrcRT_Development
```

Comment:
This won't install headers unless you use `FILE_SET` (and then you need to install the file set).

Reply:
I thought that the `PUBLIC_HEADER` property set on `orc-rt-headers` above would cause `install(TARGET ...)` to install the headers:

```cmake
set_property(TARGET orc-rt-headers
    PROPERTY PUBLIC_HEADER ${ORC_RT_HEADERS}
)
```

If I configure and build on linux with
```bash
% cmake -GNinja -DCMAKE_INSTALL_PREFIX=/path/to/install -DCMAKE_BUILD_TYPE=Debug -DLLVM_ENABLE_RUNTIMES=orc-rt /path/to/llvm-project/runtimes
% ninja install
```
I see the headers in the expected location. Am I relying on non-portable behavior here though?

Reply:
D'oh! Yes, `PUBLIC_HEADER` should work; I had forgotten about this mechanism. I believe it flattens the hierarchy, but maybe that's not the case.